src/services/EdFi.DataManagementService.Api/Program.cs

```diff
@@ -22,15 +23,23 @@
 app.MapGet("/", () => Results.Ok("Data Management Service"));
 app.MapGet("/ping", () => Results.Ok(DateTime.Now));
 
-app.MapGet("/LogAppSettings", (IOptions<AppSettings> options, ILogger<Program> logger) =>
-{
-    if (logger.IsDebugEnabled())
+app.MapGet(
+    "/LogAppSettings",
+    (IOptions<AppSettings> options, ILogger<Program> logger) =>
     {
-        logger.LogInformation($"Log information while debugging");
+        if (logger.IsDebugEnabled())
+        {
+            logger.LogInformation($"Log information while debugging");
+        }
+        logger.LogInformation(message: JsonSerializer.Serialize(options.Value));
+        return Results.Ok("Successfully logged Application Settings");
     }
-    logger.LogInformation(message: JsonSerializer.Serialize(options.Value));
-    return Results.Ok("Successfully logged Application Settings");
-});
+);
```

Comment:
@CSR2017 is it safe to remove the `/LogAppSettings` endpoint now, since it has served its temporary purpose?

Reply:
Removed